daphne_worker/src/config.rs

```diff
@@ -264,16 +253,21 @@ impl<D> DaphneWorkerConfig<D> {
         DurableConnector::new(&self.ctx.as_ref().expect("no route context configured").env)
     }
 
-    pub(crate) fn get_hpke_receiver_config_for(
+    pub(crate) fn kv(&self, binding: &str) -> Result<KvStore> {
+        self.ctx
+            .as_ref()
+            .ok_or_else(|| Error::RustError(format!("Unrecognized binding: {}", binding)))?
```

Comment:
Is "unrecognized binding" the right feedback here? What might have happened if we got an error here?

Reply:
Updated with the correct error message.